src/oaklib/interfaces/basic_ontology_interface.py

```diff
@@ -1419,7 +1419,10 @@ def entity_metadata_map(self, curie: CURIE) -> METADATA_MAP:
         raise NotImplementedError
 
     def entities_metadata_statements(
-        self, curies: Iterable[CURIE], predicates: Optional[List[PRED_CURIE]] = None
+        self,
+        curies: Iterable[CURIE],
+        predicates: Optional[List[PRED_CURIE]] = None,
+        include_nested_metadata=False,
```

Comment:
Add `**kwargs` here

Reply:
For some reason I thought I saw that it already existed. Sorry ...added.